Libplanet/State/Account.cs

```diff
@@ -0,0 +1,50 @@
+using System.Security.Cryptography;
+using Bencodex.Types;
+
+namespace Libplanet.State
+{
+    /// <summary>
+    /// An implementation of <see cref="IAccount"/> interface.
+    /// </summary>
+    public class Account : IAccount
+    {
+        public Account(
+            Address id,
+            string? memo,
+            HashDigest<SHA256> stateRootHash)
+        {
+            Id = id;
+            Memo = memo;
+            StateRootHash = stateRootHash;
+        }
+
+        public Account(Bencodex.Types.List serialized)
+        {
+            Nonce = (Integer)serialized[0];
+            Id = new Address(serialized[1]);
+            StateRootHash = new HashDigest<SHA256>((Binary)serialized[2]);
+            Memo = serialized.Count > 3 ? (Text?)serialized[3] : null;
+        }
+
+        public long Nonce { get; }
+
+        public Address Id { get; }
+
+        public string? Memo { get; }
+
+        public HashDigest<SHA256> StateRootHash { get; }
+
+        public IValue Serialize()
+        {
+             var list = Bencodex.Types.List.Empty
+                .Add(Id.Bencoded)
+                .Add(StateRootHash.ByteArray);
```

Comment:
```suggestion
             var list = new Bencodex.Types.List.List(Id.Bencoded, StateRootHash.ByteArray);
```

Also, I think `Bencodex.Types.List` might be a little bit verbose, despite its intention (i.e., differ from `List<T>`). 🤔  (we may need to discuss as other issue than this PR).

Reply:
I agree `Bencodex.Types.List` is too verbose but most of our code base is already written like that. Could we talk about this code convention on the other issue?